Lib/ast.py

```diff
@@ -613,6 +614,21 @@ def __exit__(self, exc_type, exc_value, traceback):
     def block(self):
         return self._Block(self)
 
+    @contextmanager
+    def delimit(self, delimiter):
+        """A context manager for preparing the source for expressions. It adds
```

Comment:
Hummmm, what about `start=` and `end=`. I don't like that delimiter has to be a 2-character string (there will be no failures if I pass "(HELLO)", for example).

Reply:
Yea, I think that is better too.